Development

Flyingtux opened the pull request:
Please try to solve tile download problem seen arround Saint-Malo at zoom level 17 with Bing as explained here : 
https://www.aerofly.com/community/forum/index.php?thread/14193-aeroscenery-releases-easily-create-photoreal-scenery/&postID=77365#post77365

Thanks in advance,
Jean-Pierre

Commit:
Resolved potential issue when hex names are fixed on upgrade

nickhod committed:
Changed version and change log

nickhod committed:
Work on adding new orthophoto sources

nickhod committed:
Work on additional orthophoto sources

nickhod committed:
Work on more orthophoto providers

nickhod committed:
Work on additional orhtophoto sources

Commit:
Added support for gzipped downloads

nickhod committed:
Lantmateriet orthophoto source working

nickhod committed:
Norge i Bilder orthophoto provider now working

nickhod committed:
Work on Linz orthophoto source

nickhod committed:
Linz source now takes key set in settings
Warning shown when trying to use USGS over zoom level 16

nickhod committed:
Updated changelog

nickhod committed:
Fixed issue with failed USGS tile downloads by retrying in the event of a non-success HTTP status code

nickhod commented:
Thanks. I think you've made the change against master though, so it's trying to bring in a whole load of other commits.
Would you be able to make the change against the current development branch?

Flyingtux commented:
Hi,
Actually, I didn't try yet any change yet against master or dev branch since I'm not familliar with c#. ?
I could try eventually, if I now where to put breakpoint to analyse the header of the anwser, or size of the file or color of the tile or ... Do you have an idea of the nature of the server answer in case of white tile ?

nickhod committed:
Added support for Here WeGo maps orthophoto source

Commit:
Temporary fix for HereWeGo maps version change
USGS source working again
Added GuleSiderOrthophotoSource
Add option to disable confirmation message when running multiple GeoConvert concurrently

nickhod committed:
Updating changelog

randomVRguy commented:
Anyone still active here or is this project dead ?
